src/core/gatherer.ts

```diff
@@ -46,12 +48,26 @@ export class Gatherer {
         Gatherer.browser = await chromium.launch(playwrightOptions);
       }
     }
-    const context = await Gatherer.browser.newContext(playwrightOptions);
+
+    const context = await Gatherer.browser.newContext({
+      ...playwrightOptions,
+      userAgent: await Gatherer.getUserAgent(),
+    });
+
     const page = await context.newPage();
+
```

Comment:
nit: can we get rid of these empty spaces? both in tests and code? would like to keep it similar to existing code.

Reply:
i really like spaced out code :D

Reply:
I see 😄